README.md

```diff
@@ -19,3 +19,17 @@ python3 -m unittest tests
 # Run a specific test
 python3 -m unittest tests/test_basic
 ```
+
+Badges:
```

Comment:
Can you put the badges at the top of the page? Right after the title `# crashsimilarity`.